src/java/io/bazel/rulesscala/scalac/ScalacWorker.java

```diff
@@ -243,6 +243,17 @@ private static String[] getPluginParamsFrom(CompileOptions ops) {
             "-P:dependency-analyzer:unused-deps-ignored-targets:"
                 + String.join(":", ignoredTargets));
       }
+    } 
+    
+    if (ops.enableSemanticDb) {
+
+      String[] params = {
+        "-Yrangepos",
+        "-P:semanticdb:failures:error",
+        "-Xplugin-require:semanticdb"
```

Comment:
these ops could be part of sematicdb phase. Eg.
```python
def phase_semanticdb(ctx, p):
    ...

        return struct(
            enabled = enable_semanticdb, 
            scalacopts = [
                "-Yrangepos",
               "-P:semanticdb:failures:error",
               "-Xplugin-require:semanticdb"
          ]
        )
     ...
```

then this phase can be consumed by compile phase to augment other scalacopts